base/trace_event/trace_log.cc

```diff
@@ -745,10 +745,12 @@ TraceLog::TraceLog(int generation)
       use_worker_thread_(false) {
   CategoryRegistry::Initialize();
 
+#if !defined(STARBOARD)
 #if BUILDFLAG(IS_NACL)  // NaCl shouldn't expose the process id.
   SetProcessID(0);
 #else
   SetProcessID(GetCurrentProcId());
```

Comment:
Does this fix the issue? `GetCurrentProcId` is implemented in `process_starboard.cc` and hardcoded to return 1, so I'm confused how this would fix it
Unless the issue is that we always want the process id to be the uninitialized value of 0? Though it seems to me if that's the case the deeper issue is with an missing/incorrect initialization process somewhere